app/controllers/saml_idp_controller.rb

```diff
@@ -0,0 +1,191 @@
+require 'saml_idp_constants'
+require 'saml_idp'
+require 'uuid'
+
+# rubocop:disable ClassLength
+class SamlIdpController < ApplicationController
+  include SamlIdp::Controller
+  include SamlIdpLogoutConcern
+
+  skip_before_action :verify_authenticity_token
+
+  before_action :disable_caching
+  before_action :validate_saml_request, only: :auth
+  before_action :validate_saml_logout_param, only: :logout
+  before_action :store_sp_data, only: :auth
```

Comment:
same question as [this one](https://github.com/18F/identity-idp/pull/8/files#r61797424), why do we not need this for the `:register` path?